the-collab-lab/tcl-12-smart-shopping-list
# Swap the moving pony giph for a static image of Margaret Hamilton.

## Description


The giph of the pony presented an #a11y problem. I swapped it out with the classic photo of Margaret Hamilton.

## Related Issue



N/A

## Acceptance Criteria


The image is acceptable and conveys the desired message.

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|   | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|  ✓  | :scroll: Docs              |

## Updates

### Before
![Screen Shot 2020-09-21 at 7 45 06 PM](https://user-images.githubusercontent.com/7929227/93839853-10051b80-fc43-11ea-845f-17aa9a89d2d0.png)




### After
![Screen Shot 2020-09-21 at 7 45 30 PM](https://user-images.githubusercontent.com/7929227/93839865-15626600-fc43-11ea-8733-5b43650ff044.png)




## Testing Steps / QA Criteria


Check out the [Project Brief](https://github.com/the-collab-lab/tcl-12-smart-shopping-list/blob/sb-change-project-brief-image/PROJECT-BRIEF.md) and see if you like it.
